src/Cron_Event_Command.php

```diff
@@ -153,13 +156,13 @@ public function list_( $args, $assoc_args ) {
 	 */
 	public function schedule( $args, $assoc_args ) {
 
-		$hook = $args[0];
-		$next_run = \WP_CLI\Utils\get_flag_value( $args, 1, 'now' );
+		$hook       = $args[0];
+		$next_run   = \WP_CLI\Utils\get_flag_value( $args, 1, 'now' );
```

Comment:
You've added a `use` statement for `WP_CLI\Utils` at the top of the file, so there is no need for this function call (and the one on the next line) to be fully qualified anymore.

Reply:
Changed to `Utils\get_flag_value` to use the function directly, I think we would have to use the function the itself in import, I have updated other instances too.